### ISSUE TITLE
POI Best Practice

### ISSUE THREAD

**Issue by cmheazel** [opengeospatial/poi#146]
It has been proposed that the POI Users Guide should be re-cast as a Best Practice.  The following is from the OGC TC Policies and Procedures.

### 7.4.1. Submission of documents to be considered as an OGC Best Practice or Community Practice

In order to be considered for approval as an OGC BP or CP, the document submitters shall provide the following.

- An Abstract or Introduction in the document explaining why a submitted document is relevant to the OGC.

- Evidence of implementation. Evidence of implementation shall include but not be limited to: implementation in commercial product, implementation in open source applications or software, and/or implementation in deployed applications. A single research related implementation is not proper evidence of implementation.

Approval of a BP or CP is as follows.

- Submit for TC and Public Request for comment of 30 days.

- Post the document to OGC Pending Documents on the OGC Members Portal for at least three weeks prior to the face-to-face or remote presentation.

- Present the contents of the proposed document in a TC Meeting or webinar.

**Comment by howardtrickey** [comment]
It feels like the need for commercial or open source application
implementations may be the sticking point here.

**Comment by cperey** [comment]
Yes. And, we are working on getting those…We have captured on the draft POI Wikipedia page that Digital Flanders and MyGeoVerse have implemented. 

Do we have a test suite set up to confirm their compliance?

We may have a chicken and egg situation: in order to get commercial or open source implementations, we need a "strong" user guide.
